.github/workflows/tests.yml

```diff
@@ -55,6 +57,8 @@ jobs:
 
   # Dummy step to gate other tests on without repeating the whole list
   linting-done:
+    permissions:
+      contents: none
```

Comment:
This step doesn't even do anything; what is there to compromise?

Kind of confused about the threat model you have in mind, I guess — what is this protecting against?

Reply:
It was good practice to defend any updates in the future. That was the thought process